tests_integration/test_nativeapp.py

```diff
@@ -564,3 +564,35 @@ def test_nativeapp_init_deploy(
                 env=TEST_ENV,
             )
             assert result.exit_code == 0
+
+
+# Executing "snow app teardown --cascade", verifying that the application and application objects owned by the application are dropped
+@pytest.mark.integration
+def test_nativeapp_teardown_cascade(
```

Comment:
Considering the length of the file, consider splitting the native apps integration tests up into multiple files.